lib/src/util/react_wrappers.dart

```diff
@@ -277,7 +274,8 @@ ReactElement cloneElement(ReactElement element, [Map props, Iterable children])
   }
 }
 
-/// Returns the native Dart component associated with a React JS component instance, or null if the component is not Dart-based.
+/// Returns the native Dart [ReactComponent.dartComponent] associated with a [ReactComponent] instance,
```

Comment:
`ReactComponent.dartComponent` refers to an API that's on a react-dart branch and not yet merged in.